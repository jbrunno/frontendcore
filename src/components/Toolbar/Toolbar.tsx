import React from 'react';
import {
  Toolbar as MuiToolbar,
  ToolbarProps as MuiToolbarProps,
} from '@mui/material';

export type ToolbarProps = MuiToolbarProps;

export function Toolbar(props: ToolbarProps) {
  return <MuiToolbar {...props} />;
}

export default Toolbar;
