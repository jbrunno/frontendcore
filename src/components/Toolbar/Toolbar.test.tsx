import React from 'react';
import { render } from '@test-utils/component-testing';
import { Toolbar } from './Toolbar';

test('renders the Toolbar', () => {
  const { container } = render(<Toolbar />);
  expect(container).toMatchSnapshot();
});
