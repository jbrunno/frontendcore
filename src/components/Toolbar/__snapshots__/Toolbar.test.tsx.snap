// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Toolbar 1`] = `
.emotion-0 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-left: 1rem;
  padding-right: 1rem;
  min-height: 56px;
}

@media (min-width:600px) {
  .emotion-0 {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media (min-width:0px) {
  @media (orientation: landscape) {
    .emotion-0 {
      min-height: 48px;
    }
  }
}

@media (min-width:600px) {
  .emotion-0 {
    min-height: 64px;
  }
}

<div>
  <div
    class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular emotion-0"
  />
</div>
`;
